tap-agent/src/tap/sender_account.rs

```diff
@@ -48,7 +48,14 @@ impl Inner {
         loop {
             notif_value_trigger.notified().await;
 
-            self.rav_requester_single().await;
+            if let Err(error) = self.rav_requester_single().await {
+                // If an error occoured, we shouldn't retry right away, so we wait for a bit.
```

Comment:
I think the code for doing the actual waiting is missing? (sleep?)

Reply:
we don't need to sleep (at least for testing) because it will wait for it to be notified again by other receipt.

But indeed we could add a sleep backoff in case of a high rate of receipts.